book.bib

```diff
@@ -1397,3 +1397,49 @@ @Article{ZIBB2018
 }
 
 
+
+@article{yoon2019microbial,
+  title={Microbial networks in SPRING-Semi-parametric rank-based correlation and partial correlation estimation for quantitative microbiome data},
```

Comment:
Should the title be in double curly brackets so that the capitalizations will show correctly? {{}}